### PR TITLE
385 vpc stratified by multiple variables

### PR DESCRIPTION
**Pull request by A2P2**

### CODE REVIEW

File: R/assertions.R [hunk 1]
```diff
@@ -32,6 +32,10 @@ is_string_or_formula_or_null <- function(x) {
   is_string(x) || is.formula(x) || is.null(x)
 }
 
+is_character_or_formula_or_null <- function(x) {
```
Comment (mattfidler) [inline R/assertions.R:35]: I think you should add a comment on why `is_character_or_formula_or_null` is different than `is_string_or_formula_or_null`.

Note that it accepts multiple length characters and `NA_character_` types as well (and probably `NA` which will convert to character?).

Comment (mattfidler) [reply R/assertions.R:35]: Looking at the code where this is used, you may also want to add `any(is.na(x))`  to this test.

**Review by mattfidler — COMMENTED** [review]
Small comments.

File: R/assertions.R (hunk 1 above)
Comment (A2P2) [reply R/assertions.R]: Added the `or_na` to the test. I don't see the point of `any(is.na(x))`, since `is.character(c('A', NA))` returns `TRUE`. Did you have something specific in mind when suggesting it?

Comment (mattfidler) [reply R/assertions.R]: This handles the case `is.character(NA)` which returns `FALSE` while  `is.character(c('A', NA))` returns `TRUE`.  This simply handles an edge case.  It is also found in some of the R functions that test for similar things.

What you have works fine and is OK with me too.
